pnr/openroad/meta.yaml

```diff
@@ -0,0 +1,59 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('openroad-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: openroad
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/The-OpenROAD-Project/OpenROAD.git
+    git_rev: master
+    patches:
+      - odb-drt-disable-tests.patch
+      - sta-add-install-options.patch
+      - gui-find-qt-only-if-enabled.patch
+
+build:
+  # number: 202202031935
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20220203_1935
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+
+requirements:
+  build:
+    - cmake
+    - swig
+    - bison
+    - doxygen
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
```

Comment:
@maliberty is the package functional without `cudd`? https://github.com/The-OpenROAD-Project/OpenSTA/blob/8627c6826d752ed864faafb71461d3c24ae66655/CMakeLists.txt#L394

Reply:
It doesn't seems required per: https://github.com/The-OpenROAD-Project/OpenSTA/blob/2bc6e8f68ccff771559610413a8e2f57c40f9eff/README.md#build

> CUDD is a binary decision diageram (BDD) package that is used to improve conditional timing arc handling. OpenSTA does not require it to be installed.